### PR TITLE
fix(HYS_*): drop latest calculated value (inconsistent)

### LINKED ISSUE #131: [BUG] Prepare HYS_ files to be fixed
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

last value is negative

### Expected Behavior

drop the last value, always

### Steps To Reproduce

by pre-processing pipeline

### Environment

```markdown
any
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by sbancal**
FIX #131

### CODE REVIEW

File: backend/preprocessing/hysteresis_loops.py [hunk 1]
```diff
@@ -148,8 +146,9 @@ def main():
                         columns=["n_cycles", "hysteresis_area", "stiffness", "creep"]
                     )
 
-                    experiment_metadata.N_fail = 100
+                    experiment_metadata.N_fail = 100  # TODO FIXME!
```
Comment (guilbep) [inline backend/preprocessing/hysteresis_loops.py:149]: @sbancal Is it okay to have a TODO fixme ? (should have a link to an issue) ? or do you track it in another way ?

File: backend/preprocessing/hysteresis_loops.py [hunk 2]
```diff
@@ -148,8 +146,9 @@ def main():
...
                     fill_hyst_df(df, experiment_metadata, hyst_df)
+                    hyst_df = hyst_df[:-1]  # Drop last line (it is invalid)
```
Comment (guilbep) [inline backend/preprocessing/hysteresis_loops.py:151]: Thanks for the @sbancal comment! Do you think adding an explanation on why it is invalid would be useful ?

File: backend/preprocessing/hysteresis_loops.py [hunk 3]
```diff
@@ -115,7 +113,7 @@ def PolyArea(x, y):
     Stiffness.insert(0, np.nan)
 
     # CREEP
-    ### Creep computation
+    # Creep computation
```
Comment (guilbep) [inline backend/preprocessing/hysteresis_loops.py:116]: Creep

Comment (guilbep) [reply backend/preprocessing/hysteresis_loops.py:116]: The comments below seem a bit weird. Is it documentation ? Is it dead code ? Should it help me understand better how strain should be computed ?
Where does the math come from ? Maybe I missed out on some documentation, but I would have like to have a link at least to a reference implementation or paper explaining the math. Please enlighten us! 

OOw, I got my answer in the header: 'Fatigue_test_dashboard/Hysteresis_loops.py (written by Scott)'

Soo.. What's the Fatigue_test_dashboard ? and Who's Scott ?

**Review by guilbep — APPROVED** [review]

File: backend/preprocessing/Makefile [hunk 4]
```diff
@@ -1,2 +1,3 @@
 run:
 	poetry run python3 tst_preprocessing.py --save
+	poetry run python3 hysteresis_loops.py
```
Comment (blueur) [inline backend/preprocessing/Makefile:3]: Prefer simply "python" instead of "python3" ?

File: backend/preprocessing/hysteresis_loops.py (hunk 1 above)
Comment (blueur) [reply backend/preprocessing/hysteresis_loops.py:149]: The code is from Scott, I'm wondering if it is relevant to rewrite/clean it...

Comment (sbancal) [reply backend/preprocessing/hysteresis_loops.py:149]: @guilbep good idea, I'll reference issue #136 

@blueur yes. One day ... (?)

File: backend/preprocessing/hysteresis_loops.py (hunk 3 above)
Comment (sbancal) [reply backend/preprocessing/hysteresis_loops.py:116]: I really don't know. It's _magic code_ received from the lab.

File: backend/preprocessing/Makefile (hunk 4 above)
Comment (sbancal) [reply backend/preprocessing/Makefile:3]: On Ubuntu Linux, `python` executable doesn't exist anymore. It's `python3`.
On the other hand, inside `poetry`, `python` exist.

I prefer to stay with `python3` that is more precise.

**Pull request merged**